data.sql

```diff
@@ -43,3 +45,89 @@ VALUES ('Boarmon', '06-07-2005', '7', 'true', '20.4');
 INSERT INTO animals 
 (name, date_of_birth, escape_attempts, neutered, weight_kg) 
 VALUES ('Blossom', '10-13-1998', '3', 'true', '17');
+
+-- for owners table
+
+INSERT INTO owners 
+(full_name, age) 
+VALUES ('Sam Smith', 34);
+
+INSERT INTO owners 
+(full_name, age) 
+VALUES ('Jennifer Orwell', 19);
+
+INSERT INTO owners 
+(full_name, age) 
+VALUES ('Bob', 45);
+
+INSERT INTO owners 
+(full_name, age) 
+VALUES ('Melody Pond', 77);
+
+INSERT INTO owners 
+(full_name, age) 
+VALUES ('Dean Winchester', 14);
+
+INSERT INTO owners 
+(full_name, age) 
+VALUES ('Jodie Whittaker', 38);
+
+-- species
+
+INSERT INTO species 
+(name) 
+VALUES 
+('Pokemon');
+
+INSERT INTO species 
+(name) 
+VALUES 
+('Digimon');
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith') 
+WHERE name = 'Agumon';
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell') 
+WHERE name = 'Pikachu';
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell') 
+WHERE name = 'Gabumon';
+
+-- Bob owns Devimon and Plantmon.
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') 
+WHERE name = 'Devimon';
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') 
+WHERE name = 'Plantmon';
+
+-- Melody Pond owns Charmander, Squirtle, and Blossom.
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') 
+WHERE name = 'Charmander';
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') 
+WHERE name = 'Squirtle';
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') 
+WHERE name = 'Blossom';
+
+-- Dean Winchester owns Angemon and Boarmon.
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') 
+WHERE name = 'Angemon';
+
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') 
+WHERE name = 'Boarmon';
+
+
```

Comment:
- [ ] `animals` table should be modified such that if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead

Reply:
DONE